### PR TITLE
Edge video controls resize fix

### LINKED ISSUE #2887: Attempting to change volume results in video resizing - Edge
Trying to change volume on a intergrated video results in the video being expanded in size instead of volume changing.

Exists in 4.7.0 14316

https://www.reddit.com/r/RESissues/comments/4emddq/bug_adjusting_volume_for_native_html5_video_in/

### PR DESCRIPTION
**Pull request by BenMcGarry**
Fix attempt for #2887. Build expected to fail.

Edit (erikdesjardins): Fixes #2887.

### CODE REVIEW

File: lib/modules/showImages.js [hunk 1]
```diff
@@ -1893,7 +1893,12 @@ function mousedownMedia(e) {
 		if (!e.shiftKey) {
 			if (e.target.tagName === 'VIDEO' && e.target.hasAttribute('controls')) {
 				const rc = e.target.getBoundingClientRect();
-				// ignore drag if click is in control area (40 px from bottom of video)
+				// ignore drag if click is in control area (40 px from bottom of video, 250 px for edge.)
+				if (process.env.BUILD_TARGET === 'edge') {
```
Comment (erikdesjardins) [inline lib/modules/showImages.js]: How about using
```js
const controlsHeight = (process.env.BUILD_TARGET === 'edge') ? 250 : 40;
```

**Comment by BenMcGarry** [comment]
Fix confirmed for #2887. Edge volume is now usable without resizing although it means you cannot resize the video from the bottom half, only above where the top of the volume controls being. Due to the controls being vertical this is the best fix option at this time.

File: lib/modules/showImages.js [hunk 2]
```diff
@@ -1893,8 +1893,10 @@ function mousedownMedia(e) {
 		if (!e.shiftKey) {
 			if (e.target.tagName === 'VIDEO' && e.target.hasAttribute('controls')) {
 				const rc = e.target.getBoundingClientRect();
-				// ignore drag if click is in control area (40 px from bottom of video)
-				if ((rc.height - 40) < (e.clientY - rc.top)) {
+				// ignore drag if click is in control area (40 px from bottom of video, 50% of player size for edge.)
+				const playerHalfHeight = (50 / 100) * rc.height;
+				const controlsHeight = (process.env.BUILD_TARGET === 'edge') ? playerHalfHeight : 40;
```
Comment (erikdesjardins) [inline lib/modules/showImages.js]: Nit: might as well inline it, the comment should make it clear enough
```js
? (0.5 * rc.height) : 40
```

**Comment by erikdesjardins** [comment]
Review note: although this is not ideal, I'm accepting it because:

a. There doesn't seem to be a better way
b. Few videos actually use the native controls (only direct links to video files, IIRC)

**Pull request merged**